src/App.tsx

```diff
@@ -1,109 +1,25 @@
 import React from 'react';
-
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import usersFromServer from './api/users';
+import commentsFromServer from './api/comments';
+import { PreparedPost } from './types/PreparedPost';
```

Comment:
agree
Add this file to the GitHub
<img width="588" alt="image" src="https://user-images.githubusercontent.com/91411692/232410414-4a940249-8e55-4aa5-90b8-429b71503518.png">

Reply:
done